playwright.config.ts

```diff
@@ -1,4 +1,15 @@
 import { defineConfig, devices } from '@playwright/test';
+import dotenv, { type DotenvPopulateInput } from 'dotenv';
+import { join } from 'node:path';
+import { readCanisterIdsFromJSONFile } from './shared.utils';
+
+dotenv.populate(
+	process.env as DotenvPopulateInput,
```

Comment:
the type cast is a must?

Reply:
This cast is needed to align global env type (where each value is `string | undefined`) with the dotenv.populate function param (`DotenvPopulateInput`, where each value is `string`). It should be safe to cast, as even if some var doesn't have a value in an env file, it still resolves to an empty string in `process.env`.

Reply:
Alright. Thanks for the feedback.